lib/pgbackups-archive/job.rb

```diff
@@ -13,8 +13,10 @@ def self.call
   end
 
   def initialize(attrs={})
+    @app = attrs.fetch(:app, ENV['PGBACKUPS_APP'])
```

Comment:
With the removal of the `args.with_defaults` lines from the rake task, `fetch` won't work here (since the arg keys will be found, but they'll be set to nil if no args are passed to the rake task).  Traditional hash lookups do the trick:

```ruby
@app      = attrs[:app] || ENV["PGBACKUPS_APP"]
@database = attrs[:database] || ENV["PGBACKUPS_DATABASE"] || "DATABASE_URL"
```